flight/Libraries/sanitycheck.c

```diff
@@ -146,16 +144,13 @@ int32_t configuration_check()
 				}
 				break;
 			case MANUALCONTROLSETTINGS_FLIGHTMODEPOSITION_STABILIZED1:
-				ManualControlSettingsStabilization1ThrustGet(&thrust_mode);
-				error_code = (error_code == SYSTEMALARMS_CONFIGERROR_NONE) ? check_stabilization_settings(1, multirotor, thrust_mode) : error_code;
+				error_code = (error_code == SYSTEMALARMS_CONFIGERROR_NONE) ? check_stabilization_settings(1, multirotor) : error_code;
```

Comment:
These are looooong lines. I wonder if it wouldn't be easier to read with if/else rather than ternary?

Reply:
I hate ternary :P  Still question whether it should be done this PR

Reply:
Well, it's never gonna happen unless we fix it while we're working on it IMO. :stuck_out_tongue:

Reply:
Yah, I always try to make at least some things better in each PR, but ... if I were to fix everything, we'd get too much diff :P